src/ui/src/app/chart-details/chart-details-info/chart-details-info.component.ts

```diff
@@ -26,6 +26,21 @@ export class ChartDetailsInfoComponent implements OnInit {
     return this.chart.attributes.sources || [];
   }
 
+  get sourceUrl(): string {
+    var chartSource = this.chart.attributes.repo.sourceURL;
+    if (!chartSource) return
+
+    // Used to handle possible trailing URLs
+    var urljoin = require('url-join');
```

Comment:
shouldn't requires be at the top? (not familiar with the way we're doing this though)